src/main/webapp/feed.html

```diff
@@ -4,6 +4,28 @@
     <meta charset="UTF-8" />
     <title>Melting Pot</title>
     <link rel="stylesheet" href="style.css" />
+    <script src="scripts/script.js"></script>
   </head>
-  <body></body>
+  <body>
+    <!-- Login form -->
+    <div id="login-form">
+      <label for="email">Email</label>
```

Comment:
Do we need to have Login on the Feed page too? Also for expandability it would be nice if the login form didn't need to be repeated.

Reply:
Yeah, I think having a designated login page would be a good idea.